lib/keyword-highlighter.js

```diff
@@ -27,7 +27,7 @@ function wrapper (match) {
  * @returns {String} text - the text with all highilights.
  */
 function highlight(keywords, text) {
-  var matcher = new RegExp(keywords.split(' ').join('|'), 'gi');
+  var matcher = new RegExp(keywords.split(' ').filter(k => k !== '').join('|'), 'gi');
```

Comment:
remember that ***arrow functions*** are ***not supported*** in Safari and Internet Explorer ... https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Browser_compatibility
![js-arrow-functions-not-supported-in-safari-or-internet-explorer](https://cloud.githubusercontent.com/assets/194400/12006241/579b87f4-abc6-11e5-950f-878388498557.png)